salt/kingfisher/collect/incremental.sls

```diff
@@ -47,6 +47,7 @@ include:
       'identifier': 'MOLDOVA',
       'spider': 'moldova',
       'start_date': '2021-06-11',
+      'options': '-a compile_releases=true',
```

Comment:
PR is "fix for Chile" but this modifies Moldova?

Reply:
Sorry, I edited the file directly through GitHub's interface and missed that, fixed now